### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S *.eps,*.ts,./qucs-test,./qucsator,./qucs/contrib/ps2sp -L abov,alph,ba,breal,cleare,datas,dout,fo,gir,inout,ist,manualy,nd,pard,te,ue,varn,vectore`

### CODE REVIEW

File: qucs-doc/tutorial/getstarted/content.tex [hunk 1]
```diff
@@ -2,7 +2,7 @@
 % Tutorial -- Getting Started with Qucs
 %
 % Copyright (C) 2007 Stefan Jahn <stefan@lkcc.org>
-% Copyright (C) 2007 Juan Carlos Borr�s <jcborras@gmail.com>
+% Copyright (C) 2007 Juan Carlos Borr�s <jcborras@gmail.com>
```
Comment (ckoegler) [inline qucs-doc/tutorial/getstarted/content.tex:5]: I am not sure about the character coding in tex files, but this change should be reversed.

File: qucs-doc/tutorial/lna/content.tex [hunk 2]
```diff
@@ -12,17 +12,17 @@
 
 \bigskip
 
-The main difference between as you should know, between PA and LNA, is that in the design of a LNA the noise factor is crucial, and therefore a trade off has to be made with the gain design. This design rule is well explained in all RF courses, so I will go straightforward to the solution by explaining the "pie" but not the "recipie" !
+The main difference between PA and LNA, is that in the design of a LNA the noise factor is crucial, and therefore a trade off has to be made with the gain design. This design rule is well explained in all RF courses, so I will go straightforward to the solution by explaining the "pie" but not the "recipe" !
```
Comment (ckoegler) [inline qucs-doc/tutorial/lna/content.tex:15]: Better:
The main difference, as you should know,  between PA and LNA, .....

File: qucs-doc/tutorial/modelbjt/content.tex [hunk 3]
```diff
@@ -161,9 +161,9 @@ \section*{warning}
...
 
-Philips�  fifth  generation double poly silicon wideband technology uses a steep emitter doped profile resulting in transition frequencies over 20 GHz, and with poly base contacts a low base resistance is obtained. Via the buried layer, the collector contact is brought out at the top of the die. The substrate is connected directly to the emitter package lead, resulting in improved thermal performance ( see fig \ref{design:pa:model:fifthGen}).
+Philips�  fifth  generation double poly silicon wideband technology uses a steep emitter doped profile resulting in transition frequencies over 20 GHz, and with poly base contacts a low base resistance is obtained. Via the buried layer, the collector contact is brought out at the top of the die. The substrate is connected directly to the emitter package lead, resulting in improved thermal performance ( see fig \ref{design:pa:model:fifthGen}).
```
Comment (ckoegler) [inline qucs-doc/tutorial/modelbjt/content.tex:166]: Delete cryptic character

File: qucs-doc/tutorial/opamp/content.tex [hunk 4]
```diff
@@ -789,7 +789,7 @@
...
... times obtained with discrete transistor level models\footnote{See Fig.~\ref{fig:opamp3}. Tests show that the Boyle macromodel reduces simulation times for common amplifier, timer and filter circuits by a factor between six and ten.}, while maintaining acceptable functional properties and simulation accuracy. The Boyle macromodel was designed to model differential gain versus frequency, DC common-mode gain, device input and output characteristics, slew rate limiting, output voltage swing and short-circuit limiting.  The circuit schematic for the Boyle macromodel of a bipolar OP AMP  is illustrated in Fig.~\ref{fig:opamp45}. This model consists of three connected stages: the input stage, the intermediate voltage gain stage and the output stage. Calculation of individual component values is complex, relying on a set of equations derived from the physical properties of the semiconductor devices and the structure of the electrical network.  These equations are derived in the Boyle paper and summarised in the following list. Starting with $I_{S1} = $8.0e-16, the emitter base leakage current of transistor T1, and by assuming $R2 = 100k$ the model component values can be calculated using:
```
Comment (ckoegler) [inline qucs-doc/tutorial/opamp/content.tex:792]: There seems to have been nothing changed here?

File: qucs-doc/tutorial/spicetoqucs/content.tex [hunk 5]
```diff
@@ -55,7 +55,7 @@
...
...UI simulate command (F2 key). SPICE netlist files can be linked to a Qucs SPICE netlist schematic symbol.\footnote{The schematic symbol SPICE netlist can be found in the file components section of the components icon lists on the left hand side of the GUI. Its connection pin list may be setup and edited via the Edit SPICE component properties dialogue.} These in turn can be connected, on a schematic, to any other appropriate Qucs component symbol or user defined symbol. Figure~\ref{fig:stoq_fig2} shows the resulting schematic for the two stage BJT circuit.  In this diagram the external voltage sources and amplifier load have been added together with the usual Qucs icons for DC and AC simulation of the circuit. During simulation Qucs treats the SPICE netlist component as a subcircuit\footnote{Hence the need to separate the external voltage sources and amplifier load from the main amplifier circuit.} and generates the appropriate Qucs netlist code. For example, the netlist shown in Fig.~\ref{fig:stoq_fig3} illustrates the Qucs style netlist code for the two stage BJT amplifier. Simulation of the two stage BJT amplifier gives the output waveforms displayed in Fig.~\ref{fig:stoq_fig4}.
```
Comment (ckoegler) [inline qucs-doc/tutorial/spicetoqucs/content.tex:58]: No change here too?

File: qucs/qucs-filter/qf_poly.cpp [hunk 6]
```diff
@@ -214,7 +214,7 @@ qf_double_t & qf_poly::operator [] (int i) {
   return rts[i];
 }
 
-// Returns d� (order) of polynom
+// Returns d� (order) of polynom
```
Comment (ckoegler) [inline qucs/qucs-filter/qf_poly.cpp:217]: a lot of cryptic character replacements in this file. better leave it as it is.

**Review by ckoegler — CHANGES_REQUESTED** [review]
Some lines should be changed again more carefully. In doubt, don't change them. Its just grammar.
Thanks!

**Comment by luzpaz** [comment]
Sorry, vscode is doing funky things with Unicode. I also gitkraken which also uses electron so I don't actually see these changes in the UI. I'll fix this soon

**Pull request closed without merging**